lib/simplifyExpression/stepThrough.js

```diff
@@ -1,32 +1,32 @@
 'use strict';
 
-const clone = require('./util/clone');
-
-const breakUpNumeratorSearch = require('./breakUpNumeratorSearch');
-const collectAndCombineSearch = require('./collectAndCombine/collectAndCombineSearch');
-const distributeSearch = require('./distributeSearch');
-const evaluateArithmeticSearch = require('./evaluateArithmeticSearch');
-const evaluateFunctionsSearch = require('./evaluateFunctions/evaluateFunctionsSearch');
-const multiplyFractionsSearch = require('./multiplyFractionsSearch');
-const simplifyBasicsSearch = require('./simplifyBasics/simplifyBasicsSearch');
-const simplifyDivisionSearch = require('./simplifyDivisionSearch');
-const simplifyFractionsSearch = require('./simplifyFractions/simplifyFractionsSearch');
-
-const flattenOperands = require('./util/flattenOperands');
-const hasUnsupportedNodes = require('./hasUnsupportedNodes');
-const removeUnnecessaryParens = require('./util/removeUnnecessaryParens');
-const NodeStatus = require('./util/NodeStatus');
-const print = require('./util/print');
+const checks = require('../checks');
+const Node = require('../Node');
+
+const arithmetic = require('./arithmetic');
+const basics = require('./basics');
+const division = require('./division');
```

Comment:
this is almost sorted except division

(I added a thing to the linter issue so the linter can tell us about require sorting ^_^)